tests/core/arch-arm64/arm64stalker.c

```diff
@@ -0,0 +1,652 @@
+/*
```

Comment:
This file has quite a few style issues:
- Missing space after `*` for pointer type declarations.
- Blank line before variable declarations.
- Double blank lines.
- Missing space between cast and expression being cast.
- Misaligned block alignment/style.
- Declaration after statement.